system/ee/legacy/libraries/Filemanager.php

```diff
@@ -2721,8 +2721,12 @@ public function _do_image_processing($redirect = true)
     /**
      * Image crop
      */
-    public function _do_crop($file_path)
+    public function _do_crop($file_path, $filesystem = null)
     {
+        $filesystem = ($filesystem) ?: ee('Filesystem');
+        $source = $filesystem->copyToTempFile($file_path);
+        $new = $filesystem->createTempFile();
```

Comment:
It is unnecessary on the local filesystem but it doesn't have a big performance hit and standardizes the code.  We're doing the same thing in other areas when we switched to Flysys.  It would be nice to refactor and avoid this down the road.